docs/framework/wcf/feature-details/partial-trust-feature-compatibility.md

```diff
@@ -132,14 +132,14 @@ manager: "erikre"
 -   CardSpace  
   
 -   <xref:System.IO.Log>  
-  
--   <xref:System.ServiceModel.Internal.TransactionBridge>  
+<!--
+-   System.ServiceModel.Internal.TransactionBridge  -->
   
  The following members of the <xref:System.Diagnostics.TraceOptions> enumeration should not be specified:  
   
 -   <xref:System.Diagnostics.TraceOptions>  
   
--   <xref:System.Diagnostics.TraceOptions.ProcessID>  
+-   <xref:System.Diagnostics.TraceOptions.ProcessId>  
```

Comment:
This should be ?displayProperty=fullName, as should any other enumeration members that are listed in the bulleted list.